pkgs/development/python-modules/sphinxcontrib-excel-table/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, sphinx
+, openpyxl
+, nose
+}:
+
+buildPythonPackage rec {
+  pname = "sphinxcontrib-excel-table";
+  version = "1.0.8";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256:1q79byn3k3ribvwqafbpixwabjhymk46ns8ym0hxcn8vhf5nljzd";
+  };
+
+  buildInputs = [ nose ];
+
+  propagatedBuildInputs = [ sphinx openpyxl ];
```

Comment:
There are more [requirements](https://github.com/hackerain/sphinxcontrib-excel-table/blob/master/setup.py#L40-L44). Are those not needed?

Reply:
looks like it works without them - I suspect they come in transitively with `sphinx`. Would you prefer me to add them?